pkg/dipper/errorHandling.go

```diff
@@ -16,7 +16,7 @@ func (e Error) Error() string {
 	return string(e)
 }
 
-// Is check if a given error matches this error.
+// Is : check whether the given error matches this error.
```

Comment:
question (maybe a dumb one) - does this do something different from `errors.Is` from the builtin errors package?